ui/shared/utils/constants.js

```diff
@@ -483,12 +483,22 @@ export const VEP_GROUP_SV_NEW = 'new_structural_variants'
 
 const VEP_SV_TYPES = [
   {
-    description: 'A large deletion',
+    description: 'A large deletion called from exome data',
+    text: 'gCNV Deletion',
+    value: 'gCNV_DEL',
+  },
+  {
+    description: 'A large duplication called from exome data',
+    text: 'gCNVDuplication',
+    value: 'gCNV_DUP',
+  },
```

Comment:
The text fields are not so unified. Should the `gCNVDuplication` need a blank in it? According to issue #2085, should we use `exome` instead of `gCNV` for the text fields?

Reply:
we talked offline and agreed "gCNV" was clearer than "exome". But good catch on the typo!